hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSConfigToCSConfigConverter.java

```diff
@@ -194,6 +199,10 @@ public void testDefaultUserLimitFactor() throws Exception {
 
     assertNull("root.users user-limit-factor should be null",
             conf.get(PREFIX + "root.users." + USER_LIMIT_FACTOR));
+    assertEquals("root.users auto-queue-creation-v2.enabled", "true",
+            conf.get(PREFIX + "root.users.auto-queue-creation-v2.enabled"));
+    assertNull( "root.users auto-create-child-queue.enabled should be null",
+            conf.get(PREFIX + "root.users.auto-create-child-queue.enabled"));
```

Comment:
Can you please add one testcase for checking the user-limit-factor in a conversion where the original auto-create-child-queue.enabled is true? Because currently it's not validated, so the test will pass if that check is removed.

Reply:
In class TestFSQueueConverter, there is method  testQueueWithNoAutoCreateChildQueue which checks wheather .auto-create-child-queue.enabled is null or not but according to above comment I need to add  a property of auto-create-child-queue.enabled is true.
So do I need to add a another set of queues where auto-create-child-queue.enabled is true or is this check really required?.
```
testQueueWithNoAutoCreateChildQueue() {
    converter = builder
        .withCapacitySchedulerConfig(csConfig)
        .build();

    converter.convertQueueHierarchy(rootQueue);

    assertNoValueForQueues(ALL_QUEUES, ".auto-create-child-queue.enabled",
        csConfig);
  }
```

Reply:
Yes please. The basic purpose of a test like this would be to check if the auto-create-child-queue.enabled is true the user-limit-factor = -1 isn't added.